lib/commands/plugin/create-plugin.ts

```diff
@@ -0,0 +1,100 @@
+import * as path from "path";
+import { isInteractive } from "../../common/helpers";
+
+export class CreatePluginCommand implements ICommand {
+	public allowedParameters: ICommandParameter[] = [];
+	public userMessage = "What is your GitHub username?\n(will be used to update the Github URLs in the plugin's package.json)";
+	public nameMessage = "";
+	constructor(private $options: IOptions,
+		private $errors: IErrors,
+		private $terminalSpinnerService: ITerminalSpinnerService,
+		private $logger: ILogger,
+		private $pacoteService: IPacoteService,
+		private $fs: IFileSystem,
+		private $childProcess: IChildProcess,
+		private $prompter: IPrompter) { }
+
+	public async execute(args: string[]): Promise<void> {
+		const pluginRepoName = args[0];
+		const pathToProject = this.$options.path;
+		const selectedPath = path.resolve(pathToProject || ".");
+		const projectDir = path.join(selectedPath, pluginRepoName);
+
+		this.$logger.printMarkdown("Downloading the latest version of NativeScript Plugin Seed...");
+		await this.downloadPackage(projectDir);
+
+		this.$logger.printMarkdown("Executing initial plugin configuration script...");
+		await this.setupSeed(projectDir, pluginRepoName);
+
+		this.$logger.printMarkdown("Solution for `%s` was successfully created.", pluginRepoName);
+	}
+
+	public async canExecute(args: string[]): Promise<boolean> {
+		if (!args[0]) {
+			this.$errors.fail("You must specify the plugin repository name.");
+		}
+
+		return true;
+	}
+
+	private async setupSeed(projectDir: string, pluginRepoName: string): Promise<void> {
+		const config = this.$options;
+		const spinner = this.$terminalSpinnerService.createSpinner();
+		const cwd = path.join(projectDir, "src");
+		try {
+			spinner.start();
+			await this.$childProcess.exec("npm i", { cwd: cwd });
```

Comment:
Instead of using childProcess.exec, you can use the `$npm`:
https://github.com/NativeScript/nativescript-cli/blob/master/lib/node-package-manager.ts#L18

For example:
```
await this.$npm.install(cwd, cwd);
```